storage/store.go

```diff
@@ -643,6 +644,10 @@ func (s *Store) BootstrapRange() error {
 	if err := engine.MVCCPutProto(batch, ms, key, now, nil, zoneConfig); err != nil {
 		return err
 	}
+	// Range Tree setup.
+	if _, _, err := s.SetupRangeTree(batch, ms, now, desc.RaftID); err != nil {
```

Comment:
Ah, it's a hold over from when I was going to call it from multiple places.  I'll remove it on my next commit.